### ISSUE TITLE
"k" key doesn't work while teleoperating Spot

### ISSUE THREAD

**Issue by sskorol** [SoftServeSAG/spot_simulation#9]
Hi @tarasborov,

I followed the [README]() to launch Gazebo, Spot, inverse, quadruped controller and teleop.
On the teleop tab I have the following instructions:

![Screenshot from 2021-12-10 11-03-41](https://user-images.githubusercontent.com/6638780/145547698-2b64c4ed-2eee-4f94-ad81-e51ad0081beb.png)

When I tried playing with moving around and holo features, I noticed the Spot doesn't react on "k" / "K" keys. No movements at all. The other keys seem to be working as expected.

BTW, I also noticed the "move forward" -> "I" key is more like a tiptoe. Is there any key to walk forward in a normal way? As backward movement seems more natural.

Thanks

**Comment by tarasborov** [comment]
K is not involved in a movement control. You can play with speed, body height position, etc.

**Comment by sskorol** [comment]
@tarasborov but it's present in the CLI instruction, isn't it?

![Screenshot from 2021-12-10 11-22-45](https://user-images.githubusercontent.com/6638780/145550177-662f6404-8e7e-4639-9ccc-292021c2fffb.png)


I also checked https://github.com/SoftServeSAG/teleop_legged_robots/blob/main/src/teleop_legged_robots.py#L56-L73, and there's no "k" key. So probably it'd make sense to remove it from the CLI's help to avoid confusion.

**Comment by tarasborov** [comment]
You are right. However, on the other hand, we tried to have the same CLI as common teleop has: https://github.com/ros-teleop/teleop_twist_keyboard

**Comment by sskorol** [comment]
But I don't see "k" bindings in this repo as well: https://github.com/ros-teleop/teleop_twist_keyboard/blob/master/teleop_twist_keyboard.py#L40-L59

**Comment by tarasborov** [comment]
I agree with you. We will take it into account in the next release.

**Comment by sskorol** [comment]
I filed the same issue in their repo as well: https://github.com/ros-teleop/teleop_twist_keyboard/issues/28

Thanks

**Comment by sskorol** [comment]
Feel free to close, when it's released.

**Comment by sskorol** [comment]
Seems like in the original repo they treat "k" as a "stop" action. However, it seems confusing, as there's a note that all the other keys (that are not listed in instructions) behave as a "stop" operation. So for me, it makes no sense to mention it somewhere. I'd rather assign "move back" action to "k", as it would seem more natural if you got used to classic arrow keys.
